wordpress-mobile/WordPress-iOS
# Add options in app settings to check size of media cache and to empty it.

Fixes #6252 

To test:
 - Open the app
 - Go to a blog with a lot of media
 - Create or edit a post and open the media picker
 - Select the WordPress library
 - Scroll app and down
 - Discard the post
 - Go to Me -> App Settings
 - Check the size displayed on the media cache size row
 - Press the button to Clean Media Cache
 - Check if the size changed
 - Go back to the blog, post and open the WordPress Media library again, and check if everything loads again.

Needs review: @astralbodies
